src/server/logarchiver.cpp

```diff
@@ -42,6 +44,8 @@ bool LogArchiver::getLogDirEstimatedSize(uint64_t &size, IoError &ioError) {
     for (int i = 0; i < 2; i++) {  // Retry once in case a log file is archived/created during the first iteration
         result = IoHelper::getDirectorySize(logPath, size, ioError);
         if (ioError == IoErrorSuccess) {
+            size *= 0.8;  // The compressed logs will be smaller than the original ones. We estimate at worst 80% of the
+                                  // original size.
             return true;
```

Comment:
This value seems really empirical. Should we really display a number that we know for sure to be wrong? Why not simply display the real size of the log folder before compression?

Reply:
[7d59351](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/7d593517e22955ee4f1ec0090383dea6d3bd1ede)

Reply:
[7d59351](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/7d593517e22955ee4f1ec0090383dea6d3bd1ede)